foxy/distribution.yaml

```diff
@@ -3237,6 +3237,23 @@ repositories:
       url: https://gitlab.com/ros-tracing/ros2_tracing.git
       version: foxy
     status: developed
+  ros2_usb_camera:
+    doc:
+      type: git
+      url: https://github.com/klintan/ros2_usb_camera.git
+      version: 0.1.0
```

Comment:
The `doc` and `source` entries should point to a branch, not a tag.

Reply:
it's a branch now, do I need to remove the tag as well?

Reply:
I think we should avoid branch names that are the same as tag names. I recommend creating a tag (`0.1.1`) at a point in history on the development branch (`foxy-devel`). Then, the `source` and `doc` entry would be `foxy-devel`, and then re-run bloom updating the development branch to `foxy-devel`. E.g. 

```
bloom-release --rosdistro foxy --track foxy ros2_usb_camera --edit
```